warehouse/query-core/src/main/java/datawave/query/tables/RangeStreamScanner.java

```diff
@@ -313,8 +312,7 @@ protected int scannerInvariant(final Iterator<Entry<Key,Value>> iter) {
                         
                         stats.addValue(info.count());
                         
-                        if (currentQueue.size() <= shardsPerDayThreshold
```

Comment:
Since we are not actually removing the functionality yet, move the changes in this class to you change which will rework the index lookup altogether.

Reply:
Can do